frontend/src/search-charts.ts

```diff
@@ -0,0 +1,189 @@
+import {LitElement, html} from 'lit';
+import {
+  customElement,
+  state,
+  property,
+  queryAssignedNodes,
+} from 'lit/decorators.js';
+
+import {SearchaliciousResultCtlMixin} from './search-results-ctl';
+
+import {SearchResultEvent} from './events';
+// @ts-ignore eslint-disable-next-line @typescript-eslint/ban-ts-comment  no-var
+declare var vega: any;
```

Comment:
I'm not sure of my comment proposal but we have to explain it.

Also why not use a const ? (better explain why in a comment)

Couldn't we avoid this by using `document.vega` instead of just `vega` ?

```suggestion
// Vega library is available as a global variable
// @ts-ignore eslint-disable-next-line @typescript-eslint/ban-ts-comment  no-var
declare var vega: any;
```